app/views/book_suggestions/new.html.erb

```diff
@@ -0,0 +1,64 @@
+
+<%= form_tag("/api/v1/book_suggestions") do %>
+  Autor: <%= text_field(:book_suggestion, :author) %><br>
+  Titulo: <%= text_field(:book_suggestion, :title) %><br>
+  Publicador: <%= text_field(:book_suggestion, :publisher) %><br>
+  Editorial: <%= text_field(:book_suggestion, :editorial) %><br>
+  Año: <%= text_field(:book_suggestion, :year) %><br>
+  Link: <%= text_field(:book_suggestion, :link) %><br>
+  Precio: <%= number_field(:book_suggestion, :price) %><br>
+  <button type="submit">Crear</button>
+<% end %>
+<hr>
+<table id="suggestionsTable" style="width:60%">
```

Comment:
you need to also list all the previous BookSuggestions AND add the new one when  you create one